### PR TITLE
mapper supports device data writing

### LINKED ISSUE #5657: Mapper supports device data writing
**What would you like to be added/modified**:

Added device writing feature in Mapper Framework

**Why is this needed**:

Now the Mapper Framework has realized the ability to collect and report edge device data, but it cannot well support the scenario of device data writing. In actual use, users may need to write data to certain registers of the managed device to achieve the purpose of device control.

### PR DESCRIPTION
**Pull request by wbc6080**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:

Now the Mapper Framework has realized the ability to collect and report edge IoT device data to user applications and user databases. but it cannot well support the scenario of device data writing. 
This PR is based on the object model definition and implements the `method` field in device-instance to describe the methods that the device can be called by the outside world, thereby completing the purpose of device data writing. For more detailed architecture, please refer to https://github.com/kubeedge/kubeedge/pull/5662

**Which issue(s) this PR fixes**:

Fixes #5657 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: staging/src/github.com/kubeedge/mapper-framework/pkg/common/datamethod.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package common
```
Comment (Shelley-BaoYue) [inline staging/src/github.com/kubeedge/mapper-framework/pkg/common/datamethod.go:1]: add copyright here.

Comment (wbc6080) [reply staging/src/github.com/kubeedge/mapper-framework/pkg/common/datamethod.go:17]: Done

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Review by fisherxu — COMMENTED** [review]
/assign @WillardHu @JiaweiGithub

**Pull request merged**